webgoat-container/src/main/resources/templates/main_new.html

```diff
@@ -84,20 +84,21 @@
                         <span th:text="${@environment.getProperty('webgoat.build.number')}"></span></a></li>
                 </ul>
             </div>
+            <div style="display:inline" id="reset_db">
+                <a href="#resetLessonDatabase">
+                    <button type="button" id="reset-db-button" class="btn btn-default right_nav_button button-up"
+                            th:title="#{database.reset}">
+                        <i class="fa fa-refresh"></i>
```

Comment:
The "refresh" look and feel seems to friendly to me. Intuitively I would not expect a reset of all my lessons.

Reply:
I will add the text reset database for all lessons? it is in the general menu
The idea is that the reset lesson button would only reset the current lesson database. This is a button to go back to the original state, does that make sense?

Reply:
Hmm maybe I do not understand what a reset does. It is not the same as restart lesson. `Is it just intended for future upgrades where the local database could be different then the expected one?

Reply:
it was a feature request from some while ago during a workshop. But you are right, if we have the reset lesson functionality which resets the DB this one becomes a bit confusing. Closing the PR